extension/content_dashboard.js

```diff
@@ -7,16 +7,73 @@ $(function() {
     airline = AES.getAirlineCode();
     server = AES.getServerName();
     chrome.storage.local.get(['settings'], function(result) {
-        settings = result.settings;
+      settings = result.settings;
```

Comment:
Can I ask why you changed the indenting?

Reply:
that was not intentional